cmd/agent/subcommands/run/internal/clcrunnerapi/v1/clcrunner.go

```diff
@@ -68,3 +69,25 @@ func flattenCLCStats(stats status.CLCChecks) map[string]status.CLCStats {
 
 	return flatened
 }
+
+func getCLCRunnerWorkers(w http.ResponseWriter, r *http.Request) {
+	log.Info("Got a request for the runner workers")
+	w.Header().Set("Content-Type", "application/json")
+	stats, err := status.GetExpvarRunnerStats()
+	if err != nil {
+		log.Errorf("Error getting exp var stats: %v", err)
+		body, _ := json.Marshal(map[string]string{"error": err.Error()})
+		http.Error(w, string(body), 500)
+		return
+	}
+
+	jsonWorkers, err := json.Marshal(stats.Workers)
```

Comment:
should we use instead our own structure to send the data. in never the expvar runner stats change?

Reply:
I checked `GetExpvarRunnerStats()` and in fact it is already the case.